src/pages/VenuePage/VenuePage.tsx

```diff
@@ -63,6 +64,8 @@ const VenuePage: React.FC = () => {
   // const [isAccessDenied, setIsAccessDenied] = useState(false);
 
   const { user, profile } = useUser();
+  const { userLocation } = useWorldUserLocation(user?.uid);
+  const { lastSeenIn: userLastSeenIn } = userLocation ?? {};
```

Comment:
I mentioned this somewhere else, but maybe the object desctructure can be removed to simplify this.